src/include/os_fhandle.i

```diff
@@ -156,6 +160,7 @@ static inline int
 __wt_write(WT_SESSION_IMPL *session,
     WT_FH *fh, wt_off_t offset, size_t len, const void *buf)
 {
+	struct timespec start, stop;
```

Comment:
I'm not certain thats true. I see plenty of code where we declare the timespec's before the macro types. Example src/txn/txn_ckpt.c, src/btree/bt_io.c-, src/btree/bt_read.c and others. In functions in these files timespec is the first variable declared, when it is declared. Perhaps we let this one go through to "the keeper"?